packages/browser/examples/single/bundle/src/App.js

```diff
@@ -28,11 +28,11 @@ import {
   handleIncomingRedirect,
   fetch,
   getDefaultSession,
-} from "../../../../dist/index";
+} from "@inrupt/solid-client-authn-browser";
 
 const REDIRECT_URL = "http://localhost:3113/";
-const CLIENT_APP_WEBID =
-  "https://raw.githubusercontent.com/inrupt/solid-client-authn-js/main/packages/browser/examples/single/bundle/client-app-profile.ttl#app";
+const CLIENT_IDENTIFIER =
```

Comment:
Might be worth documenting this a bit

Reply:
14f6fc32dfd73e4562f67811e7e42011b051305f